test/dtypes.c

```diff
@@ -205,7 +211,7 @@ user_compound_convert(hid_t H5_ATTR_UNUSED src_id, hid_t H5_ATTR_UNUSED dst_id,
 static int
 test_classes(void)
 {
-    struct complex { /* Struct with complex fields */
```

Comment:
Another struct that had to be renamed so it doesn't conflict with the "complex" keyword.